### DISCUSSION TITLE
Cyberduck:  Transfers window empty

### DISCUSSION THREAD

**Discussion by protectiveit** [iterate-ch/cyberduck#15378] [Q&A] [answered]
I'm performing an upload of an 8TB file structure to Wasabi and I noticed that my Transfers window is empty and everything is grayed out even though I've got a transfer in progress and can see Cyberduck going through the files one by one down in the bottom status bar of the main Cyberduck window.

The uploads appear to be working correctly however I cannot stop/resume the transfers or tell the current completion percentage.  I'm using version 8.7.1.  This particular install is on Windows Server 2022 Standard.

Thanks for any assistance you can provide.

**Comment by dkocher** [comment, marked as answer]
It looks you have checked [Transfers → General → Browser connection for file transfers](https://docs.cyberduck.io/cyberduck/transfer/#transfers-general-browser-connection-for-file-transfers) or same _Transfer Files_ setting in the [bookmark](https://docs.cyberduck.io/cyberduck/bookmarks/#bookmark-options).

**Reply by protectiveit** [reply]
Thanks for the assistance!  I'm not sure how it got set but it was the browser connection setting in the bookmark that was causing it.
